pkg/manager/impl/validation/validation.go

```diff
@@ -121,6 +145,22 @@ func ValidateNamedEntityIdentifierListRequest(request admin.NamedEntityIdentifie
 	return nil
 }
 
+func ValidateNamedEntityListRequest(request admin.NamedEntityListRequest) error {
+	if err := ValidateEmptyStringField(request.Project, shared.Project); err != nil {
+		return err
+	}
+	if err := ValidateEmptyStringField(request.Domain, shared.Domain); err != nil {
+		return err
+	}
+	if err := ValidateResourceType(request.ResourceType); err != nil {
+		return err
+	}
+	if request.Limit == 0 {
```

Comment:
nit: you can use ValidateLimit https://github.com/lyft/flyteadmin/blob/master/pkg/manager/impl/validation/validation.go#L184